cmd/finch/virtual_machine_remove_test.go

```diff
@@ -49,19 +52,28 @@ func TestRemoveVMAction_runAdapter(t *testing.T) {
 				logger.EXPECT().Info(gomock.Any()).AnyTimes()
 			},
 		},
+		{
+			name: "should forcibly remove the instance",
+			cmd:  removeCmd,
+			args: []string{
+				"--force",
+			},
+			mockSvc: func(logger *mocks.Logger, creator *mocks.LimaCmdCreator, ctrl *gomock.Controller) {
+				command := mocks.NewCommand(ctrl)
+				creator.EXPECT().CreateWithoutStdio("remove", "--force", limaInstanceName).Return(command)
+				command.EXPECT().CombinedOutput()
+				logger.EXPECT().Info(gomock.Any()).AnyTimes()
+			},
+		},
 	}
 
+	//nolint:paralleltest // run in serial because tests with different args can interfere with each other
```

Comment:
Can you add more details into how tests with different arguments interfere with each other? Shouldn't that not be the case if we mock everything correctly?

Reply:
I just realized that this is just because I use the same instance of the cobra command. will fix in next revision